Makefile

```diff
@@ -147,6 +147,18 @@ endif
 	CC=$(CC) go install -a -v -tags "$(TAG_SQLITE3)" $(DEBUG) ./...
 	@echo "LXD built successfully"
 
+race:
+ifeq ($(TAG_SQLITE3),)
+	@echo "Missing custom libsqlite3, run \"make deps\" to setup."
+	exit 1
+endif
+
+	go get -t -v -d ./...
+	CC=$(CC) go install -race -v -tags "$(TAG_SQLITE3)" $(DEBUG) ./...
+	CGO_ENABLED=0 go install -v -tags netgo ./lxd-p2c
```

Comment:
@stgraber race detector doesn't work without cgo, so cant enable on these two steps.

Reply:
Yeah, guess we should just skip on those then